experimenter/experimenter/experiments/migrations/0266_nimbusexperiment_results_pairwise_data.py

```diff
@@ -0,0 +1,165 @@
+# Generated by Django 3.2.23 on 2024-01-24 21:10
+
+from copy import deepcopy
+from django.db import migrations, models
+
+
+def update_results_data_schema(apps, schema_editor):
+    NimbusExperiment = apps.get_model("experiments", "NimbusExperiment")
+
+    for experiment in NimbusExperiment.objects.all():
+        branches = [b.slug for b in experiment.branches.all()]
+        reference_branch = experiment.reference_branch
+        data = experiment.results_data
+        if data is not None and "v3" not in data and "v2" in data:
+            data["v3"] = data["v2"]
```

Comment:
We can do the not-continue thing here too to de-nest one level.

```suggestion
        if data is None or "v3" in data or "v2" not in data:
            continue
            
        data["v3"] = data["v2"]
```